pkg/controllers/managementuser/rbac/globalrolebinding_handler.go

```diff
@@ -181,38 +186,41 @@ func (c *grbHandler) ensureProvisioningClusterAdminBinding(obj *v3.GlobalRoleBin
 	provCluster := pClusters[0]
 
 	subject := rbac.GetGRBSubject(obj)
-	rbName := name.SafeConcatName(rbac.ProvisioningClusterAdminName(provCluster), rbac.GetGRBTargetKey(obj))
-
-	existingRb, err := c.rbLister.Get(provCluster.Namespace, rbName)
-	if err != nil && !apierrors.IsNotFound(err) {
-		return err
-	}
-	if existingRb != nil {
-		return nil
-	}
+	crtbName := fmt.Sprintf("crtb-%s-%s-%s", rbac.GetGRBTargetKey(obj), "restricted-admin", "cluster-owner")
```

Comment:
Are you sure this doesn't still need `SafeConcatName`?

Reply:
We should use a similar function here but not `SafeConcatName` since its high collision rate makes it unsafe.